Mono.Debugging.Soft/SoftDebuggerSession.cs

```diff
@@ -3382,22 +3380,7 @@ public void ApplyChanges (ModuleMirror module, byte[] metadataDelta, byte[] ilDe
 			else
 				pdbArray = rootDomain.CreateByteArray (pdbDelta);
 
-			module.Assembly.ApplyChanges_DebugInformation (metadataDelta, pdbDelta);

```

Comment:
It looks like this removal and others above are the result of a bad rebase or merge.